src/pages/ReceiverDetailsEdit.tsx

```diff
@@ -352,15 +387,23 @@ export const ReceiverDetailsEdit = () => {
                       label={
                         <InfoTooltip
                           hideTooltip={
-                            !isVerificationFieldConfirmed("YEAR_MONTH")
+                            !isVerificationFieldSubmitted("YEAR_MONTH")
+                          }
+                          showCheckMark={isVerificationFieldConfirmed(
+                            "YEAR_MONTH",
+                          )}
+                          infoText={
+                            isVerificationFieldConfirmed("YEAR_MONTH")
+                              ? alreadyConfirmedText
+                              : alreadySubmittedText
                           }
-                          infoText={alreadyConfirmedText}
                         >
                           Date of Birth (Year & Month only)
                         </InfoTooltip>
                       }
                       fieldSize="sm"
-                      value={receiverEditFields.yearMonth}
+                      type="date"
+                      max={new Date().toISOString().split("T")[0]}
```

Comment:
With type month, you need to use the slice method to set the max:
```suggestion
                      max={new Date().toISOString().slice(0, 7)}
```

Reply:
Same as above reason. Firefox and Safari don't support month type.